bindata/manifests/machine-config/files/configure-switchdev.sh.yaml

```diff
@@ -0,0 +1,51 @@
+mode: 0755
+overwrite: true
+path: "/usr/local/bin/configure-switchdev.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -eux
+    input="/etc/switchdev.conf"
+    names=()
+
+    while read pci_addr num_vfs
+    do
+      echo "Set $num_vfs VFs on device $pci_addr"
+
+      names+=($(ls /sys/bus/pci/devices/${pci_addr}/net/))
+      # create VFs
+      echo $num_vfs > /sys/bus/pci/devices/${pci_addr}/sriov_numvfs
+    done <"$input"
```

Comment:
It looks like `input` file contains cluster-wide switchdev device info. should we apply the config according to node specific config ?

Reply:
The assumption is we have an identical hardware setup for all the offload workers. All the nodes in one MachineConfigPool will share the identical MachineConfig.